src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersExtensions.cs

```diff
@@ -1,5 +1,9 @@
 namespace NServiceBus
 {
+#if NETSTANDARD2_0
```

Comment:
I'm wondering if it would just be better to have the fully-qualified types below instead of needing a separate `#if` just for using statements.